runtime/neumann/src/lib.rs

```diff
@@ -1218,7 +1219,7 @@ impl_runtime_apis! {
 			action: AutomationAction,
 			executions: u32,
 		) -> Balance {
-			AutomationTime::calculate_execution_fee(&(action.into()), executions)
+			AutomationTime::calculate_execution_fee(&(action.into()), executions).unwrap()
```

Comment:
I will need to write a new version of the RPC where you pass in the encoded call since you need to real thing to get the weight for DynamicDispatch